src/bun.js/api/bun/socket.zig

```diff
@@ -2826,7 +2826,7 @@ fn NewSocket(comptime ssl: bool) type {
             const TCPHandler = NewWrappedHandler(false);
 
             // reconfigure context to use the new wrapper handlers
-            Socket.unsafeConfigure(this.socket.context(), true, true, WrappedSocket, TCPHandler);
+            Socket.unsafeConfigure(this.socket.context().?, true, true, WrappedSocket, TCPHandler);
```

Comment:
What's the `.?` do at the end?

Reply:
it's like `!` in typescript. It says "I assert that this optional value will never be optional at this call site"

Reply:
Nice!